atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/config/TargetedSweepRuntimeConfig.java

```diff
@@ -46,9 +46,10 @@ public int shards() {
     }
 
     /**
-     * If true, we batch many iterations on each shard and strategy upon obtaining the lock. This should lead to
-     * higher throughput in targeted sweep at the expense of more uneven sweeping across different shards.
+     * @deprecated This configuration is ignored, use {@link #enableAutoTuning()} or
+     * {@link #maximumPartitionsToBatchInSingleRead()} instead.
      */
+    @Deprecated
     @Value.Default
     public boolean batchShardIterations() {
```

Comment:
I think you can delete it and put in a JsonIgnoreProperties annotation at the top.